### PR TITLE
Added several entries to psx, ibm5170_cdrom and cd32

### PR DESCRIPTION
**Pull request by RalfVB**
psx: 
Kaette Kita Cyborg Kuro-chan (Japan) [kaetkita] 
Chou Jikuu Yousai Macross - Ai Oboete Imasu ka (Japan) [macross]

ibm5170_cdrom:
Alien Rampage [arampage]
Crusader - No Remorse [crusader]
Iron Blood [ironbld]
Rise 2 Resurrection - Director's Cut [rise2dc]
Turrican II - The Final Fight [turricn2]
Zyclunt [zyclunt]

cd32:
Akira [akira]
Alfred Chicken [alfred]
Alien Breed - Tower Assault [abreedta]
Alien Breed 3D [abreed3d]
Alien Breed Special Edition + Qwak [abseqwak]
Banshee [banshee]
Bump & Burn [bumpburn]
The Chaos Engine [chaoseng]
Disposable Hero [disphero]
Fire & Ice [fireice]
The Classic Lotus Trilogy [lotus]
Project-X + Ultimate Body Blows [projxubb]
Rise of the Robots [riserobo]
Seek & Destroy [seekdest]
Shadow Fighter [shadfght]
Chambers of Shaolin [shaolin]
Super Putty [suprputt]
Total Carnage [totcarn]

### CODE REVIEW

**Comment by RalfVB** [comment]
I have no idea what you're talking about? Last psx entry I submitted is kaetkita, which works and looks fine at my end.
Could it be you're confusing this with the previous pr somehow?...
The only problem I see is that the last addition to ibm5170_cdrom has now caused a merge conflict with mine that was fine before.

**Comment by RalfVB** [comment]
Sorry about that, I removed the .save file.

**Comment by RalfVB** [comment]
Everything should be fine now. Added some new things for good measure.

File: hash/ibm5170_cdrom.xml [hunk 1]
```diff
@@ -197,7 +233,36 @@
 			<feature name="disk_label" value="CD0141" />
 			<diskarea name="cdrom">
 				<disk name="pompei" sha1="964c914b8580a21b2f1a54e9329336ead243598b" />
-			</diskarea>			
+			</diskarea>
+		</part>
+	</software>
+  
+	<software name="prototyp">
```
Comment (DopefishJustin) [inline hash/ibm5170_cdrom.xml]: prototyp was already added in one of your previous pull requests so this is a duplicate and causes a ```mame -valid``` failure

File: hash/psx.xml [hunk 2]
```diff
@@ -47366,6 +47381,28 @@ The entries in this section are intended to replace the existing "low-grade" Jap
...
+		-->
+		<description>Chou Jikuu Yousai Macross - Ai Oboete Imasu ka (Japan)</description>
+		<year>1999</year>
+		<publisher>Bandai</publisher>
+		<part name="cdrom" interface="psx_cdrom">
+			<diskarea name="cdrom">
+				<disk name="chou jikuu yousai macross - ai oboete imasu ka (japan) (disc 1)" sha1="f231291ee425f4e6044f2711bb97d2357972d630"/>
+			</diskarea>
+		</part>
+		<part name="cdrom" interface="psx_cdrom">
```
Comment (DopefishJustin) [inline hash/psx.xml]: you have two parts both called "cdrom" here which causes a ```mame -valid``` failure

File: hash/ibm5170_cdrom.xml [hunk 3]
```diff
@@ -197,7 +233,36 @@
...
+		</part>
+	</software>
+
+	<software name="rise2dc" supported="partial">
+		<description>Rise 2 Resurrection - Director's Cut</description>
+		<year>1995</year>
+		<publisher>Acclaim</publisher>
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="rise 2 resurrection - director's cut (disc 1).chd" sha1="cd62b2bde9355b2625d8bdeaece3ec2ccfff5310" />
```
Comment (DopefishJustin) [inline hash/ibm5170_cdrom.xml]: Don't include .chd in the disk name, that makes it look for ```rise 2 resurrection - director's cut (disc 1).chd.chd```

File: hash/ibm5170_cdrom.xml [hunk 4]
```diff
@@ -489,4 +565,16 @@
 			</diskarea>
 		</part>
 	</software>
+
+	<software name="zyclunt">
```
Comment (DopefishJustin) [inline hash/ibm5170_cdrom.xml]: As per your previous comment, the game doesn't work in MAME so mark it ```supported="no"```

**Comment by DopefishJustin** [comment]
Almost there, thanks for the effort so far!

File: hash/psx.xml (hunk 2 above)
Comment (RalfVB) [reply hash/psx.xml]: fixed

**Comment by RalfVB** [comment]
Everything should be good now. Feel free to let me know if it isn't.

I have to say adding software manually is a pretty laborious and error-prone task. I'm working on a tool that does all the heavy lifting (called MSLM - https://github.com/RalfVB/mslm). Very easy to use and guarantees valid, consistent output.

**Pull request merged**